bin/server_package_standalone

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+###########################################################################
+# Packaging script which creates standalone package.
+#
+# Usage:
+#> bin/server_package_standalone
+
+set -e
+
+BINARY=${BINARY:="build/server/mysterium_server"}
+DIR_BUILD="build/package"
+PACKAGE_FILE="${DIR_BUILD}/`basename -s .exe ${BINARY}`.tar.gz"
+
+# Put all files to package directory (to avoid preserving directory structure)
```

Comment:
*to avoid preserving* - did you mean *to preserve*?

Reply:
No. By default, `mkdir` preserves directory structure of global paths